### PR TITLE
Create initial Medscape style

### PR DESCRIPTION
**Pull request by jaamulberry**
This style is used by medscape https://www.medscape.com/ This is the initial commit which takes care of Journal articles

### CODE REVIEW

File: medscape [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-us">
+  <info>
+    <title>Medscape Medical Style</title>
+    <title-short>Medscape</title-short>
+    <id>https://csl.mendeley.com/styles/554082391/medscape</id>
```
Comment (jaamulberry) [inline medscape]: Not sure how to define the id without being in the repository yet.

**Comment by adam3smith** [comment]
Thanks -- could you describe how this style is going to be used? Medscape doesn't look (at first glance) like it's open for submissions, so who is going to use this style and it what context? 
Asking to see whether this fits https://github.com/citation-style-language/styles/wiki/Criteria-for-Accepting-Styles

**Comment by jaamulberry** [comment]
Hey @adam3smith,
Sorry for the delay. Just saw your reply. Medscape does allow publications (see https://www.medscape.com/public/publisherscircle). Let me know if you require additional information.

**Comment by adam3smith** [comment]
@jaamulberry -- 
Thanks, I had seen that, but that looks like an aggregation of existing publications:

> It includes a broad range of full-text material from familiar print and electronic publications including clinical journals, clinical reference books, professional periodicals, newsletters, searchable databases, self-tests, and more. 

Just to be clear, I'm perfectly happy to take this style if it's useful for a large-ish group of people, but I do want to understand who will use it and how to make sure there aren't better ways of making it available.

**Comment by jaamulberry** [comment]
@adam3smith Medscape and Web MD publish unique content for both patients, researchers, and physicians. They rely on a growing network of contributors to submit research outputs in varying forms to be published across their platforms. Their network is extensive and growing - with submissions coming from around the globe and across medical, research, and clinical fields.

Historically, they have had to share endnote style files for each submitter to use. More and more of their research collaborators are not using EndNote and would like to submit to CSL to offer more flexibility. They also are pushing contributors to provide more formal citations to streamline the submission process and ensure consistency and reproducibility.

**Comment by rmzelle** [comment]
@jaamulberry, thanks.

So is there an online style guide available for this style? Per https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link, we typically require each CSL style to include a link to the format it describes.

**Comment by rmzelle** [comment]
@jaamulberry, if there is a style guide for Medscape we can still accept this style. Is it available on the Medscape website somewhere?

**Comment by rmzelle** [comment]
Closing because of inactivity.

**Pull request closed without merging**